secret/secret_test.go

```diff
@@ -239,14 +239,14 @@ func TestCreateWithValidSecrets(t *testing.T) {
 			return "2"
 		}
 
+		// Delete will not be called in this context (rv == 2).
```

Comment:
Should we then assert that it _isn't_ called?

Reply:
Hm. Yes. Will add.

Reply:
Done with commit efdca4854e69f6cccccf77f5ba2d98b501eb2cf3